cartridges/app_storefront_training/cartridge/templates/default/home/homePage.isml

```diff
@@ -0,0 +1,29 @@
+<isdecorate template="common/layout/page">
```

Comment:
I could not find the commit with the original file, so I can see what you changed since the storefront_base.
The recommendation is, every time you copy some file from storefront_base, create a commit, then create another one with the changes.

Reply:
Please do this only with this file just for exercising purposes. Remove this new file from the project, commit this removal, add it again, commit the original copy, then place your changes and commit.
I know there are many other files like this, but do just this one.